plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/bitbucketServer.ts

```diff
@@ -174,6 +177,21 @@ export function createPublishBitbucketServerAction(options: {
             description:
               'The token to use for authorization to BitBucket Server',
           },
+          gitCommitMessage: {
+            title: 'Git Commit Message',
+            type: 'string',
+            description: `Sets the commit message on the repository. The default value is 'initial commit'`,
+          },
+          gitAuthorName: {
+            title: 'Default Author Name',
+            type: 'string',
+            description: `Sets the default author name for the commit. The default value is 'Scaffolder'`,
+          },
+          gitAuthorEmail: {
+            title: 'Default Author Email',
+            type: 'string',
+            description: `Sets the default author email for the commit.`,
+          },
```

Comment:
```suggestion
          gitAuthorName: {
            title: 'Author Name',
            type: 'string',
            description: `Sets the author name for the commit. The default value is 'Scaffolder'`,
          },
          gitAuthorEmail: {
            title: 'Author Email',
            type: 'string',
            description: `Sets the author email for the commit.`,
          },
```

Reply:
I know the other actions use the word `default` in descriptions but I think we should drop it as it's redundant.